packages/components-react/projects/remix/app/routes/modal-route-change.open.tsx

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useState } from 'react';
+import { useNavigate } from '@remix-run/react';
+import { PButton, PButtonGroup, PModal, PText } from '@porsche-design-system/components-react/ssr';
+
+export default function OverviewModal(): JSX.Element {
+  const navigate = useNavigate();
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(true);
+  const onDismiss = useCallback(() => {
+    setIsModalOpen(false);
+    navigate(-1);
+  }, []);
+
+  const style = `
+    #app {
+      transform: none;
+    }
+  `;
+
+  return (
+    <>
+      <style dangerouslySetInnerHTML={{ __html: style }} />
+      <PModal heading="Some Heading" open={isModalOpen} onDismiss={onDismiss}>
```

Comment:
Probably can always we open? Otherwise you would need to wait for the closing transition to finish before "navigating" back.